deck.gl/examples/scripting/over_under_zoom.html

```diff
@@ -0,0 +1,60 @@
+<html>
+  <head>
+    <!-- deck.gl standalone bundle -->
+    <script src="https://unpkg.com/deck.gl@^8.1.0/dist.min.js"></script>
+    <!-- TODO: remove when package is published -->
+    <script src="../../../../deck.gl/modules/carto/dist.min.js"></script>
+
+    <!-- Mapbox dependencies -->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.0/mapbox-gl.js"></script>
+    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.0/mapbox-gl.css" rel="stylesheet" />
+
+    <style type="text/css">
+      body {margin: 0; padding: 0;}
+      #map {width: 100vw; height: 100vh;}
+    </style>
+  </head>
+
+  <body>
+    <div id="map"></div>
+  </body>
+
+  <script type="text/javascript">
+
+    deck.carto.setDefaultCredentials({
+      username: 'public',
+      apiKey: 'default_public'
+    });
+
+    const deckgl = new deck.DeckGL({
+      container: 'map',
+      mapStyle: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+
+      initialViewState: {
+        latitude: 51.507,
+        longitude: -0.127,
+        zoom: 10
+      },
+      controller: true,
+
+      layers: [
+        new deck.carto.CartoSQLLayer({
+          data: `SELECT * FROM london_neighbourhoods`,
+          getLineColor: [255, 255, 255],
+          getFillColor: [130, 109, 186],
+          pointRadiusMinPixels: 6,
+          lineWidthMinPixels: 1,
+
+          // Define the zoom range of the layer
+          minZoom: 12,
+          maxZoom: 16,
+
+          // Allow underzooming by setting the bounding box of the layer.
```

Comment:
Need more clarification, something like this: by setting minZoom, maxZoom and autoExtent, the layer will be visible for zoom>16 but it will always use z=16 tiles, for zoom<12 the layer will be visible but it always use z=12 tiles

Reply:
:ok_hand: